src/config/permissions.js

```diff
@@ -0,0 +1,15 @@
+const normal = {
+  articles: ['read', 'create', 'delete', 'edit'],
+  comments: ['read', 'create', 'delete', 'edit'],
+  tags: ['read', 'create', 'delete'],
+  users: ['read', 'create', 'edit', 'delete']
+};
+
+const admin = {
+  articles: ['read', 'delete'],
+  comments: ['read', 'delete'],
```

Comment:
and on comment admin can not add a comment

Reply:
The admin cannot add a comment, he/she can only read and delete.